docs/pages/enroll-resources/database-access/enroll-aws-databases/rds-oracle.mdx

```diff
@@ -0,0 +1,413 @@
+---
+title: Database Access with AWS RDS Oracle with Kerberos Authentication
+description: How to configure Teleport Database Access with AWS RDS Oracle with Kerberos authentication.
+---
+
+(!docs/pages/includes/database-access/db-introduction.mdx dbType="Amazon RDS for Oracle" dbConfigure="with Kerberos authentication"!)
+
+## How it works
+
+The Teleport Database Service joins the same Active Directory domain as the Oracle
+database and uses the Kerberos protocol to authenticate with Oracle.
+When a user connects to Oracle via Teleport, the Database Service obtains a
+service ticket from Active Directory, then uses a long-term key for the database
+user to decrypt the ticket and connect to Oracle. At that point, the
+Database Service forwards user traffic to the database.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- An Amazon RDS for Oracle database instance.
+- An AWS Directory Service Managed Microsoft AD.
+- A Linux node joined to the same Active Directory domain as the database. This
+  guide will walk you through the joining steps if you don't have one.
```

Comment:
we should also have a pre-req on the client tool `tsh db connect` will use. unfortunately as this point we don't know which oracle clients are more popular. like if it's a particular GUI client, we can show steps for `tsh proxy db` instead